Update the instructions to disable ASM, following ENT customer feedback (#20759)
DataDog/documentation: content/en/security/application_security/threats/library_configuration.md

To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. Once it's removed, restart your service.

> In case no `DD_APPSEC_ENABLED=true` environment variable is set for your service:

Reviewer: ```suggestion
If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
```

> * If it's a PHP service: explicitely set the environment variable to `DD_APPSEC_ENABLED=false` and restart your service

Reviewer: ```suggestion
* If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false` and restart your service
```

> * If ASM was activated in one-click: go to [ASM service configuration][8], click on your service and press 'Deactivate'

Reviewer: ```suggestion
* If ASM was activated in one-click: go to [ASM service configuration][8], click on your service and then click **Deactivate**.
```
